test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxServiceTests.cs

```diff
@@ -930,102 +928,109 @@ public void VerifyExpectedAndActualBooleanOptionsDictionary(Dictionary<string, D
                 Assert.That(actualValue, Is.EqualTo(expectedValue), $"Actual Property from Service is not equal to default property for {optionRow.Key}");
             }
         }
+    }
+}
 
-        /// <summary>
-        /// Verify the generate Tsql model operation
-        /// </summary>
-        [Test]
-        public void GenerateTSqlModelFromSqlFiles()
-        {
-            DacFxService service = new DacFxService();
-            Directory.CreateDirectory(TSqlModelTestFolder);
-            string sqlTable1DefinitionPath = Path.Join(TSqlModelTestFolder, "table1.sql");
-            string sqlTable2DefinitionPath = Path.Join(TSqlModelTestFolder, "table2.sql");
-            const string table1 = @"CREATE TABLE [dbo].[table1]
+[TestFixture]
+public class TSqlModelRequestTests
+{
+    private string TSqlModelTestFolder = Path.Combine("..", "..", "..", "DacFx", "TSqlModels");
+
+    private DacFxService service = new DacFxService();
+
+    [SetUp]
+    public void Create()
+    {
+        Directory.CreateDirectory(TSqlModelTestFolder);
+    }
+
+    [TearDown]
+    public void CleanUp()
+    {
+        Directory.Delete(TSqlModelTestFolder);
+    }
+
+    /// <summary>
+    /// Verify the generate Tsql model operation
+    /// </summary>
+    [Test]
+    public void GenerateTSqlModelFromSqlFiles()
+    {
+        string sqlTable1DefinitionPath = Path.Join(TSqlModelTestFolder, "table1.sql");
```

Comment:
We should be careful when making statically-named files in a single location. I'd suggest having each test run generate a random folder name (usually using a timestamp is useful for this) instead of always using TSqlModels